contracts/human_checker/tests/workspaces.rs

```diff
@@ -171,5 +188,37 @@ async fn is_human_call() -> anyhow::Result<()> {
     tokens = suite.query_sbts(&john).await?;
     assert_eq!(tokens, None);
 
+
+    //
+    // Test Vote with lock duration
+    //
+
+    //
+    // test1: too short lock duration: should fail
+    let mut payload = VotePayload{prop_id: 10, vote: "wrong_option".to_string()};
+    let r = suite.is_human_call_lock(&john, VOTING_DURATION / 2 *3,  &payload).await?;
+    assert!(r.is_failure());
+    let failure_str = format!("{:?}",r.failures());
+    assert!(failure_str.contains("sufficient amount of time"));
+
+    //
+    // test2: second call, should not change
+    let r = suite.is_human_call_lock(&john, VOTING_DURATION / 2*3,  &payload).await?;
```

Comment:
```suggestion
    let r = suite.is_human_call_lock(&alice, VOTING_DURATION / 2*3,  &payload).await?;
```

Reply:
also the `voting_duration` is wrong it will not fail because its bigger should be `VOTING_DURATION` / (2 * 3)

Reply:
in this case the assertion needs to be changed to check for the error thrown in `is_human_call_lock`